pkg/apis/metrics/clickhouse_fetcher.go

```diff
@@ -122,6 +116,14 @@ const (
 		WHERE is_done = 0 
 		GROUP BY database, table
 	`
+
+	querySystemDisksSQL = `
+	    SELECT 
+	        name,
+            toString(free_space) AS free_space,
+			toString(total_space) AS total_space			
+        FROM system.disks
+	`
```

Comment:
@alz there is main part of backport
which affects
following picture
![image](https://user-images.githubusercontent.com/105560/96276918-0aee7e80-0fed-11eb-8909-86167fc280e2.png)